skyportal/handlers/api/internal/image_analysis.py

```diff
@@ -0,0 +1,607 @@
+from baselayer.app.access import permissions
+from baselayer.app.env import load_env
+from baselayer.log import make_log
+import os
+import arrow
+from astropy.io import fits
+from astropy.wcs import WCS
+from astropy.table import vstack
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import search_around_sky
+from astropy.time import Time
+import astropy.units as u
+
+import base64
+import numpy as np
+import sqlalchemy as sa
+import tempfile
+from tqdm.auto import tqdm
+from tornado.ioloop import IOLoop
+import shutil
+
+from ..photometry import add_external_photometry
+from ...base import BaseHandler
+from ....models import Instrument, User, DBSession
+
+_, cfg = load_env()
+
+log = make_log('image_analysis')
+
+try:
+    if cfg['image_analysis'] is True:
```

Comment:
I'm not sure why we need these "if"s. I think this whole file should not be imported or called if the image analysis is turned off. If we can assume that then we can move the imports to the beginning and get rid of a few conditionals. 
Or maybe I'm missing something?

Reply:
I did that to make sure that someone's app does not crash if they activated the features but did not install all of the dependencies, but that's a little overkill. I will implement your suggestion.

Reply:
Also, that would require more to import the config in the __init__.py file of the internal directory, so that I can conditionally import the handler. Is that alright? @guynir42

Reply:
@mcoughlin I failed to get an answer from Guy on this one. Any thoughts?

Reply:
I think we can import the config anywhere we want and it lazy loads it (if I remember correctly). I don't see why that would be a problem...

Reply:
Of course, it's not a technical problem at all. I was just asking myself if it was good practice / conventional to do so. It has been added